src/main/scala/com/yoo/app/dao/MongoStore.scala

```diff
@@ -36,6 +39,13 @@ class MongoStore(collection: MongoCollection[Document])(implicit ec: ExecutionCo
         .getOrElse(Left(LookupError(s"Image: $id does not exist")))
     }
 
+  override def getImage(id: String): Future[Either[CollectionError, Image]] =
```

Comment:
Implementation for obtaining image + base64 encoded content in `MongoStore.scala`